mobsfscan/rules/semgrep/android/hidden_ui.yaml

```diff
@@ -16,13 +16,13 @@ rules:
               $X.setVisibility($V);
     message: >-
       Hidden elements in view can be used to hide data from user. But this data
-      can be leaked.
+      can be leaked. If the view contains sensitive data, it might still be accessible through memory inspection.
     languages:
       - java
     severity: ERROR
     metadata:
       cwe: cwe-919
       owasp-mobile: m1
       masvs: storage-7
-      reference: >-
-        https://github.com/MobSF/owasp-mstg/blob/master/Document/0x05d-Testing-Data-Storage.md#checking-for-sensitive-data-disclosure-through-the-user-interface-mstg-storage-7
+      references:
```

Comment:
This will fail with the semgrep version that we use with mobsfscan.

Reply:
You are right @ajinabraham, I will improve/complete some rule descriptions without changing the reference. Thanks. Sebas (Semgrep).